src/common/misc.c

```diff
@@ -657,6 +657,12 @@ daos_crt_init_opt_get(bool server, int ctx_nr)
 	/** enable statistics on the server side */
 	daos_crt_init_opt.cio_use_sensors = server;
 
+	/** configure cart for maximum bulk threshold */
+	daos_crt_init_opt.cio_use_expected_size = true;
```

Comment:
this is a bitfield, probably better to set it to 1 instead of true, in case true expands to 32bit